core/pkg/server/server.go

```diff
@@ -55,6 +56,9 @@ type Server struct {
 
 	// loggerPath is the default logger path
 	loggerPath string
+
+	// debug is the debug flag
```

Comment:
comment not descriptive

Reply:
ah that's the stupid cursor adds code when i press on tab

Reply:
let me fix it something more useful